roles/openshift_health_checker/library/aos_version.py

```diff
@@ -47,8 +47,6 @@ def bail(error):  # pylint: disable=missing-docstring
 
     # determine what level of precision we're expecting for the version
     expected_version = module.params['version']
-    if expected_version.startswith('v'):  # v3.3 => 3.3
-        expected_version = expected_version[1:]
```

Comment:
Correct, now looking at the single commit I see it!
If we wanted a more generic module, it would be interesting if the module itself handled the version format... but that is harder to do code reuse... I wonder if Ansible always copy every file from `library/` or just the file with the same name as the module being invoked?

In any case, this module is very specific, it is fine to expect a particular format of version. Could you please document the expected format in the module, just for our future reference?